### PR TITLE
[Apache Tomcat] Add integration package with session data stream

### PR DESCRIPTION
**Pull request by milan-elastic**
- Enhancement

## What does this PR do?

- Added 1 data stream (Session metrics).
- Added data collection logic for the data streams.
- Added the ingest pipeline for the data streams.
- Mapped fields according to the ECS schema and added Fields metadata in the appropriate yaml files.
- Added dashboards and visualizations.
- Added system test cases for the data stream.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] Verify if field mapping is correct in the data stream template.
- [x] Verification of data in visualisation after enabling TSDB flag in kibana
- [x] Verification of the count of documents (before & after TSDB enablement) in Discover Interface

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/apache_tomcat) directory.
- Run the following command to run tests. elastic-package test

## Screenshots
apache_tomcat-session-dashboard

## Meta issue
- #5200

## Related issue
- #6391

## Related PRs
- #5424
- #5926
- #5883
- #5983
- #6003
- #6527 
- #6596
- #6609

### CODE REVIEW

File: packages/apache_tomcat/data_stream/session/fields/fields.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            - name: rate
+              type: double
+              description: Session expiration rate in sessions per minute.
+              metric_type: gauge
+            - name: total
+              type: double
+              description: Number of sessions that expired (doesn't include explicit invalidations).
+              metric_type: gauge
+        - name: persist_authentication
+          type: double
```
Comment (rajvi-elastic) [inline packages/apache_tomcat/data_stream/session/fields/fields.yml:65]: I think the type of this field should be boolean as per the description. Can you please verify once?

**Review by rajvi-elastic — APPROVED** [review]
LGTM!

**Review by muthu-mps — COMMENTED** [review]
By enabling TSDB for this integration, Update the Kibana version to 8.8 in all datastreams as the minimum supported version is 8.8.

**Comment by milan-elastic** [comment]
> By enabling TSDB for this integration, Update the Kibana version to 8.8 in all datastreams as the minimum supported version is 8.8.

Sure @muthu-mps , I have raised one [issue](https://github.com/elastic/integrations/issues/6735) for it. As part of it, I'll work on it and raise the PR for TSDB enablement and version bump.

File: packages/apache_tomcat/data_stream/session/sample_event.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        "namespace": "ep",
+        "type": "metrics"
+    },
+    "ecs": {
+        "version": "8.7.0"
+    },
+    "elastic_agent": {
+        "id": "16d1abf3-8caa-4fa1-b85a-bee568d095fb",
+        "snapshot": false,
+        "version": "8.7.0"
```
Comment (muthu-mps) [inline packages/apache_tomcat/data_stream/session/sample_event.json:56]: As the integration is supported from 8.8 version. Can we generate the sample metrics with `8.8` version of the stack ?

**Pull request merged**